### PR TITLE
Code Owners

### PR DESCRIPTION
**Pull request by Nikita-dev98**
# Description

This file lets everyone know who is responsible for the code and who should be reviewing changes.

### CODE REVIEW

**Comment by ghost** [comment]
Reviewing this right now, let's see...

**Review by ghost — CHANGES_REQUESTED** [review]
Oh looks like the `CODEOWNERS` file I gave you is outdated. It has **Carl listed**, but he's... no longer with the company. Could you **remove him from the file**, but not from our hearts, and push that change?

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (Nikita-dev98) [inline CODEOWNERS]: This user is removed

**Review by ghost — CHANGES_REQUESTED** [review]
Seems like the `CODEOWNERS` file is missing the right content. Just remove Carl from the file and keep the rest. Could you fix that and push the changes?

**Review by ghost — APPROVED** [review]
Approved! Go ahead and merge this PR. Quickly now.

**Pull request merged**